docs/r2/tensorboard_scalars_and_keras_basic.ipynb

```diff
@@ -0,0 +1,452 @@
+        "    loss='mse', # keras.losses.mean_squared_error\n",
+        "    optimizer=keras.optimizers.SGD(lr=0.2),\n",
+        ")\n",
+        "\n",
+        "print(\"Training ... With default parameters, this takes less than 10 seconds.\")\n",
+        "training_history = model.fit(\n",
+        "    X_train, # input\n",
+        "    Y_train, # output\n",
+        "    batch_size=train_size,\n",
+        "    verbose=0, # Suppress chatty output; use Tensorboard instead\n",
+        "    epochs=100,\n",
+        "    validation_data=(X_test, Y_test),\n",
+        "    callbacks=[tensorboard_callback],\n",
+        ")\n",
+        "\n",
+        "print(\"Average test loss: \", np.average(training_history.history['loss']))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "042k7GMERVkx",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Examining loss using TensorBoard\n",
+        "\n",
+        "Now, start TensorBoard, specifying the root log directory.\n",
+        "\n",
+        "Wait a few seconds for TensorBoard's UI to spin up. "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "6pck56gKReON",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/scalars"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ciSIRibhRi6N",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "You may see TensorBoard say that \"No dashboards are active for the current data set\". That's because initial logging data hasn't been saved yet. As training progresses, the Keras model will start logging data. TensorBoard will periodicially refresh and show you your scalar metrics. If you're impatient, you can tap the Refresh arrow on the top right.\n",
+        "\n",
+        "As you watch the training progress, note how both training and validation loss rapidly decrease and then remain stable. In fact, you could have stopped training after 25 epochs, because the training didn't improve much after that point.\n",
+        "\n",
+        "Hover over the graph to see specific data points. You can also try zooming in with your mouse, or selecting part of them to view more detail.\n",
+        "\n",
+        "Using the \"Runs\" selector on the left, you can select specific runs, or select only training or validation. As you continue to experiment and develop your model, you can compare runs by selecting only those you want to examine.\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "finK0GfYyefe",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Ok, TensorBoard indicates that the metrics look good! Now see how the model actually behaves in real life. \n",
```

Comment:
Is there anything we can say about why the metrics look good? How did we judge that?

Reply:
Reworded.